quickstarts/install-netobserv.yaml

```diff
@@ -0,0 +1,62 @@
+apiVersion: console.openshift.io/v1
+kind: ConsoleQuickStart
+metadata:
+  name: netobserv
+spec:
+  conclusion: Your Network Observability Operator is installed and configured.
+  description: Install Network Observability Operator and deploy a FlowCollector to capture traffic.
+  displayName: Observe the cluster network traffic
+  durationMinutes: 5
+  introduction: '### In this quick start, you'll deploy and configure Network Observability Operator to get an overview of your cluster network traffic including ingress and egress traffic. For more information on Network Observability, see the related [OpenShift documentation.](https://docs.openshift.com/container-platform/latest/observability/network_observability/network-observability-overview.html)'
+  tags:
+    - network
+    - observability
+    - netobserv
+  tasks:
+    - description: |-
+        You can install the Network Observability Operator using the OpenShift Container Platform web console Operator Hub. When you install the Operator, it provides the FlowCollector custom resource definition (CRD). You can set specifications in the web console when you create the FlowCollector.
+        1. In the OpenShift Container Platform web console, click **OperatorHub** from the [Operators]{{highlight qs-nav-operators}} → OperatorHub.
+        2. Choose **Network Observability Operator** from the list of available Operators in the OperatorHub, and click **Install**.
```

Comment:
Is it possible to provide a direct link to that?

Reply:
But isn't it what lightspeed is doing?
It seems that this link would work: `/operatorhub/ns/default?details-item=netobserv-operator-redhat-operators-openshift-marketplace&channel=stable` (assuming using the main official catalog)

Reply:
Does it works if you remove the channel ?

Else I feel it's not worth. [None of the existing quickstarts](https://github.com/openshift/console-operator/tree/master/quickstarts) are doing such.